maze_transformer/evaluation/plot_maze.py

```diff
@@ -6,20 +6,113 @@
 
 from maze_transformer.generation.latticemaze import LatticeMaze
 
+CoordArray = NDArray["coords", np.int8]
+
+# Set size of unit.
+# Each "unit" consists of a node and the right and lower adjacent wall/connection. Its area is ul * ul.
+# Wall thickness is fixed to 1, so this parameter sets the ratio of node length to wall thickness
+ul = 14
+
+
+# Maze conversion functions (No plt involved)
+
+
+def as_img(maze: LatticeMaze) -> NDArray["x y", bool]:
+    """
+     Build an image to visualise the maze.
+     Each "unit" consists of a node and the right and lower adjacent wall/connection. Its area is ul * ul.
+     - Nodes are displayed as white squares of area: (ul-1) * (ul-1)
+     - Walls are displayed as black recktangles of area: 1 * (ul-1)
+     - Connections are displayed as light grey or white rectangles of area: 1 * (ul-1); color is depending on show_connections argument
+
+     Axes definition:
+    (0,0)     y
+      -|----------->
+     x |
+       |
+       |
+       v
+
+     Returns a matrix of side length (ul) * n + 1 where n is the number of nodes.
+    """
+
+    # Set color of connections (using plt.imshow(cmap='grey))
+    show_connections = True
+
+    white_color = 100
+    if show_connections:
+        connection_color = white_color * 0.93
+    else:
+        connection_color = white_color
```

Comment:
`show_connections` should either be a parameter or it should be removed. I say we remove it, then we can simplify this.